frontend/src/utils/getTokens.ts

```diff
@@ -0,0 +1,21 @@
+interface Tokens {
+  accessToken: string;
+  refreshToken: string;
+}
+
+const getTokens = (): Tokens => {
+  const tokens: Tokens = {
+    accessToken: "",
+    refreshToken: "",
+  };
+  const tokensJSON = localStorage.getItem("tokens");
+  if (tokensJSON) {
+    const { accessToken, refreshToken } = JSON.parse(tokensJSON) as Tokens;
+    tokens.accessToken = accessToken;
+    tokens.refreshToken = refreshToken;
+    return tokens;
```

Comment:
Would this be cleaner?
```suggestion
    return { accessToken, refreshToken}
```